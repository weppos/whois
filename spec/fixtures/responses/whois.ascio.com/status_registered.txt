The data in Ascio Technologies' WHOIS database is provided
by Ascio Technologies for information purposes only. By submitting
a WHOIS query, you agree that you will use this data only for lawful
purpose. In addition, you agree not to:
(a) use the data to allow, enable, or otherwise support any marketing
activities, regardless of the medium used. Such media include but are
not limited to e-mail, telephone, facsimile, postal mail, SMS, and
wireless alerts; or
(b) use the data to enable high volume, automated, electronic processes
that sendqueries or data to the systems of any Registry Operator or
ICANN-Accredited registrar, except as reasonably necessary to register
domain names or modify existing registrations.
(c) sell or redistribute the data except insofar as it has been
incorporated into a value-added product or service that does not permit
the extraction of a substantial portion of the bulk data from the value-added
product or service for use by other parties.
Ascio Technologies reserves the right to modify these terms at any time.
Ascio Technologies cannot guarantee the accuracy of the data provided.
By accessing and using Ascio Technologies WHOIS service, you agree to these terms.

NOTE: FAILURE TO LOCATE A RECORD IN THE WHOIS DATABASE IS NOT
INDICATIVE OF THE AVAILABILITY OF A DOMAIN NAME.
Registrant:
  ASCIO Technologies Inc. (ASCIOTEC1364)
  Islands Brygge 55

  Copenhagen, S, 2300
  DK

  Domain name: ascio.com

Technical contact:
  Hostmaster, Netnames (NH323743)
  Group NBT
  3rd Floor Prospero House
  241 Borough High St.
  London, SE1 1GA
  GB
  hostmaster@netnames.net
  +44.2070159370 Fax: +44.2070159375

Administrative contact:
  Hostmaster, Technical (TH6437)
  Speednames
  Rejsbygade 8a

  Copenhagen, DK, 1759
  DK
  hostmaster@speednames.com
  +45.33886300 Fax: +45.33886301

Record created:           2005-03-01 14:11:50
Record last updated:      2013-06-02 00:23:21
Record expires:           2014-05-31 00:00:00

Domain servers in listed order:
  ns1.ascio.net (NSASCION521)
  ns2.ascio.net (NSASCION542)
  ns3.ascio.net (NSASCION563)
  ns4.ascio.net (NSASCION141)

Registrar Name: Ascio Technologies, Inc
Registrar WHOIS: whois.ascio.com
Registrar Homepage: http://www.ascio.com
