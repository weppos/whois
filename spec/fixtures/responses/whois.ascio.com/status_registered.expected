#domain
  %s == "ascio.com"

#domain_id
  %s == ""


#status
  %s == :registered

#available?
  %s be_falsey

#registered?
  %s be_truthy


#created_on
  %s %CLASS{time}
  %s %TIME{2005-03-01 14:11:50 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2013-06-02 00:23:21 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2014-05-31 00:00:00 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "106"
  %s.name         == "Ascio Technologies, Inc"
  %s.organization == "Ascio Technologies, Inc"
  %s.url          == "http://www.ascio.com"


#registrant_contacts
  %s %CLASS{array}
  %s.size eq(1)
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  %s[0].name          == "Hostmaster Ascio"
  %s[0].organization  == "ASCIO Technologies Inc."
  %s[0].address       == "Islands Brygge 55"
  %s[0].city          == "Copenhagen"
  %s[0].zip           == "2300"
  %s[0].state         == "S"
  %s[0].country_code  == "DK"
  %s[0].phone         == ""
  %s[0].fax           == ""
  %s[0].email         == ""

#admin_contacts
  %s %CLASS{array}
  %s.size eq(1)
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_ADMINISTRATIVE
  %s[0].name          == "Technical Hostmaster"
  %s[0].organization  == "Speednames"
  %s[0].address       == "Rejsbygade 8a"
  %s[0].city          == "Copenhagen"
  %s[0].zip           == "1759"
  %s[0].state         == "DK"
  %s[0].country_code  == "DK"
  %s[0].phone         == "+45.33886300"
  %s[0].fax           == "+45.33886301"
  %s[0].email         == "hostmaster@speednames.com"

#technical_contacts
  %s %CLASS{array}
  %s.size eq(1)
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  %s[0].name          == "Netnames Hostmaster"
  %s[0].organization  == "Group NBT"
  %s[0].address       == "3rd Floor Prospero House, 241 Borough High St."
  %s[0].city          == "London"
  %s[0].zip           == "SE1 1GA"
  %s[0].state         == ""
  %s[0].country_code  == "GB"
  %s[0].phone         == "+44.2070159370"
  %s[0].fax           == "+44.2070159375"
  %s[0].email         == "hostmaster@netnames.net"


#nameservers
  %s %CLASS{array}
  %s.size eq(4)
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.ascio.net"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.ascio.net"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.ascio.net"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns4.ascio.net"
